src/NewWindow.js

```diff
@@ -97,6 +98,9 @@ class NewWindow extends React.PureComponent {
 
     // Open a new window.
     this.window = window.open(url, name, toWindowFeatures(features))
+    if (callback) {
```

Comment:
I think this check should be better:
```suggestion
    if (typeof callback === 'function') {
```
wdty?

---
(sorry for long delay, I have been pretty busy these days)